index.html

```diff
@@ -0,0 +1,286 @@
+                </div>
+        
+                <p class="msg">
+                    A daily selection of privately 
+                    personalized reads; no accounts 
+                    or sign-ups required.
+                </p>
+        
+                <ul class="list-two flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+        
+                <button>
+                    See Project
+                </button>
+                </section>
+        
+        
+            </div>
+        
+            <!-- card 3 -->
+        
+            <!------------------------------------------------------------->
+        
+            <!-- card 4 -->
+        
+            <div class="card flex">
+        
+                <figure class="fig-image four">
+                </figure>
+        
+                <section class="content">
+                <h2 class="head-two">Multi-Post Stories</h2>
+        
+                <div class="list flex">
+                    <h3 class="head-three">CANOPY</h3>
+                    <div><img src="images/Counter.svg" alt="point"></div>
+                    <div>Back End Dev</div>
+                    <div><img src="images/Counter.svg" alt="point"></div>
+                    <div>2015</div>
+                </div>
+        
+                <p class="msg">
+                    A daily selection of privately 
+                    personalized reads; no accounts 
+                    or sign-ups required.
+                </p>
+        
+                <ul class="list-two flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+        
+                <button>
+                    See Project
+                </button>
+                </section>
+        
+        
+            </div>
+        
+            <!-- card 4 -->
+        
+            <!---------------------------------------------------------------------------->
+ 
+            <!-- About Me -->
+            <div class="wrapper">
+            
+                <section class="about">
+                <h2>About<br>Myself</h2>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+
+                <h3>LET'S CONNECT</h3>
+
+                <div class="icons flex">
+                    <div><img src="icons/space.svg" alt="icon"></div>
+                    <div><img src="icons/twitter.svg" alt="icon"></div>
+                    <div><img src="icons/chi.svg" alt="icon"></div>
+                    <div><img src="icons/git.svg" alt="icon"></div>
+                </div>
```

Comment:
- [ ] Please the list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag).